pylot/perception/tracking/tracking_eval_operator.py

```diff
@@ -44,6 +45,7 @@ def __init__(self, obstacle_tracking_stream, ground_obstacles_stream,
         self._sim_interval = None
```

Comment:
Can we add make sim_interval an optional argument for the operator, so that if it is known ahead of time (e.g. in the offline pipeline) a definitive value could be provided instead of being inferred. This would help the tracker avoid skipping the first frame inference, which is what the tracker does now in order to infer the frame gap.